surveys/2021-annual-survey/questions.md

```diff
@@ -198,16 +198,16 @@ Select all that apply:
 > specific answer appears frequently, we can add it to future surveys to reduce
 > the amount of work needed to process responses.
 
-### On the primary machine you compile Rust code on, how many logical CPU threads do you have?
+### On the primary machine you compile Rust code on, how many CPU do you have?
```

Comment:
```suggestion
### On the primary machine you compile Rust code on, how many CPUs are there?
```
Even if you decide to keep the original wording, we should use the plural of CPUs here.